src/components/TransactionForm/TransactionForm.js

```diff
@@ -0,0 +1,118 @@
+import { Router } from '@/routes';
+import './TransactionForm.scss';
+import { transactionsCollectionRef } from '@root/firebase';
+import { addDoc, getDocs, query, where } from 'firebase/firestore';
+import { categresCollectionRef } from '../../../firebase';
+
+export default function TransactionForm() {
+  this.typeOptions = ['income', 'outcome', 'transfer', 'correction'];
+  this.categoryOptions = [];
+  this.elements = {
+    form: document.createElement('form'),
+    formOwner: null,
+    formDate: new Date(),
+    formType: document.createElement('select'),
```

Comment:
this field should change the way wallet's inputs are shown.

type `income` - should show only `to` wallet
type `outcome` - only `from` wallet
type `transaction` - should show both
type `correction` - should also show only `to` wallet